models/utils/detect_face.py

```diff
@@ -5,13 +5,49 @@
 from PIL import Image
 import numpy as np
 import os
+import math
 
 # OpenCV is optional, but required if using numpy arrays instead of PIL
 try:
     import cv2
 except:
     pass
 
+def fixed_batch_processing(im_data, model, model_name):
+    """
+    Use fixed-batch size for rnet and onet to avoid out of GPU memory error
+    im_data:   input for 'rnet' or 'onet'
+    model : rnet or onet
+    model_name : 'rnet' or 'onet'
+    """
```

Comment:
Can you please use a consistent docstring format to the rest of the project?

Reply:
Use your code. docstring is not necessary.